packages/dialog/src/index.js

```diff
@@ -179,3 +201,27 @@ function useForkedRef(...refs) {
     // eslint-disable-next-line react-hooks/exhaustive-deps
   }, refs);
 }
+
+function validateAriaLabel(props, name, compName, ...rest) {
+  const details =
+    "\nSee https://www.w3.org/TR/wai-aria/#aria-label for details.";
+  if (!props["aria-label"] && !props["aria-labelledby"]) {
+    return new Error(
+      "You must provide either an `aria-label` prop or an `aria-labelledby` prop to the `" +
+        compName +
+        "` component." +
```

Comment:
Instead of saying "the \`Dialog\` component" let's say "a \<Dialog> element" or just "a \<Dialog>".

Reply:
Updated https://github.com/reach/reach-ui/pull/317/commits/ae3794dcf240bd27a68688dc3f6b461953ea3d2f